### ISSUE TITLE
Communicate over raw TCP/telnet?

### ISSUE THREAD

**Issue by probonopd** [NeoTheFox/RepRaptor#10]
In addition to USB/serial, would it be possible to communicate over raw TCP/telnet?

**Comment by NeoTheFox** [comment]
Yes, I've wanted to it for a while now, but got too busy with other work before I've had a chance to throw that in. It just requires one more Qt dependency and few inner working tweaks. So far it's on the roadmap.

**Comment by aytekno** [comment]
Hello,
Are you still working on communicating over TCP?

**Comment by NeoTheFox** [comment]
No, at this time I haven't started implementing it. The development of RepRaptor is only going when I have the free time for it sadly, as I am the only contributor right now
